local-setup/scripts/ci-dataloader-v2-regression.py

```
@@ -0,0 +1,696 @@
+    """Employee created via HRMS should have password set via user service."""
+    import time
+    import requests as req
+
+    ts = int(time.time()) % 100000
+    test_tenant = f"{ROOT_TENANT}.ciemp{ts}"
+    loader.create_tenant(test_tenant, f"CI Emp Test {ts}")
+    time.sleep(2)
+
+    # Create a user via _create_user_for_tenant (which uses egov-user directly)
+    test_username = f"CI_EMP_{ts}"
+    test_password = "eGov@123"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "test_password"

(S105)